templates/layouts/bootstrap.html.ep

```diff
@@ -30,6 +30,7 @@
           $(function() {
           setupForAll();
           %= content_for 'ready_function'
+          newFeature();
```

Comment:
Not sure if this is the appropriate way to accomplish this, but i have to somehow load the bootstrap-tour instructions when traversing through multiple pages. Example will follow soon.